airbyte-cdk/python/unit_tests/sources/declarative/requesters/error_handlers/backoff_strategies/test_constant_backoff.py

```diff
@@ -8,17 +8,23 @@
 from airbyte_cdk.sources.declarative.requesters.error_handlers.backoff_strategies.constant_backoff_strategy import ConstantBackoffStrategy
 
 BACKOFF_TIME = 10
+OPTIONS_BACKOFF_TIME = 20
+CONFIG_BACKOFF_TIME = 30
 
 
 @pytest.mark.parametrize(
-    "test_name, attempt_count, expected_backoff_time",
+    "test_name, attempt_count, backofftime, expected_backoff_time",
     [
-        ("test_exponential_backoff", 1, BACKOFF_TIME),
-        ("test_exponential_backoff", 2, BACKOFF_TIME),
+        ("test_constant_backoff_first_attempt", 1, BACKOFF_TIME, BACKOFF_TIME),
+        ("test_constant_backoff_second_attempt", 2, BACKOFF_TIME, BACKOFF_TIME),
```

Comment:
@brianjlai are you referring to the backoff time or the attempt count? I added tests for both.
1. backoff time is a float -> backoff time is evaluated as a float
2. attempt count is a float -> the value is actually never used. Python auto casts it to int. Do you think we should be stricter and fail if we pass a float instead of an int?

Reply:
i just meant for backoff time, we were using the constants for `BACKOFF_TIME` which were ints, so just adding a true float. I was messing around with things and using the constants eval would come back as int, figured we should just test a float returned too just in case. But I see you added it already so all good!